Software/grimoirelab.md

```diff
@@ -1,7 +1,56 @@
 # GrimoireLab
 
-GrimoireLab is a set of free, open source software tools for software development analytics. They gather data from many systems supporting development (git, GitHub, Jira, Bugzilla, Gerrit, mailing lists, Jenkins, Slack, Discourse, Confluence, StackOverflow, etc), merge and organize it in a database, and produce visualizations, actionable dashboards, and analytics of all of it.  GrimoireLab is focused on analyzing activity, community and processes, but can be easy tailored for other aims. They can also be easily integrated with other tools.
+GrimoireLab is a set of free, open source software tools for software development analytics. They gather data from several platforms involved in software development (Git, GitHub, Jira, Bugzilla, Gerrit, Mailing lists, Jenkins, Slack, Discourse, Confluence, StackOverflow, [and more](https://gitlab.com/Bitergia/c/FAQ/tree/master/sources#supported-data-sources)), merge and organize it in a database, and produce visualizations, actionable dashboards, and analytics of all of it.  
```

Comment:
I suggest changing the link to GrimoireLab website. 

This PR would help: https://github.com/chaoss/grimoirelab/pull/393

Reply:
ok! thanks for the feedback. Changes have been made @GeorgLink :+1: